packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.stories.js

```diff
@@ -231,3 +231,31 @@ WithTruncation.args = {
   maxLines: 2,
   maxLinesSuffix: '...'
 };
+
+export const WithStrikeThrough = args =>
+  class WithParsing extends lng.Component {
+    static _template() {
+      return {
+        InlineContent: {
+          type: InlineContentComponent,
+          x: 550,
+          w: 400,
+          contentWrap: args.contentWrap,
+          contentProperties: args.contentProperties,
+          justify: args.justify,
+          content: [
+            'Rent',
+            { newline: true },
+            {
+              text: '$19.99 ',
```

Comment:
i think a trailing space sneaked in here

```suggestion
              text: '$19.99',
```

Reply:
If the space is not added the text is too close together. If you take a look at the file most text has added a space.

Reply:
But I will remove it and push up. It may look good to you without the added space.

Reply:
I think this space is the only thing left to change.